rendercv/cli/commands.py

```diff
@@ -136,6 +144,32 @@ def cli_command_render(
     extra_data_model_override_argumets: typer.Context = None,  # type: ignore
 ):
     """Render a CV from a YAML input file."""
+
+    if watch:
+
+        def rerun_command():
+            cli_command_render(
+                input_file_name=input_file_name,
+                use_local_latex_command=use_local_latex_command,
+                output_folder_name=output_folder_name,
+                latex_path=latex_path,
+                pdf_path=pdf_path,
+                markdown_path=markdown_path,
+                html_path=html_path,
+                png_path=png_path,
+                dont_generate_markdown=dont_generate_markdown,
+                dont_generate_html=dont_generate_html,
+                dont_generate_png=dont_generate_png,
+                watch=False,
+                extra_data_model_override_argumets=extra_data_model_override_argumets,
+            )
+
+        file_path = utilities.string_to_file_path(input_file_name)
```

Comment:
In v1.14 we removed this function from `utilities`. It should be `input_file_path: pathlib.Path = pathlib.Path(input_file_name).absolute()` like line 175.